structuralcodes/materials/reinforcement/_reinforcementMC2010.py

```diff
@@ -66,3 +70,29 @@ def epsud(self) -> float:
     def gamma_eps(self) -> float:
         """The partial factor for ultimate strain."""
         return self._gamma_eps or 0.9
+
+    def __elastic__(self) -> dict:
+        """Returns kwargs for creating an elastic constitutive law."""
+        return {'E': self.Es}
+
+    def __elasticperfectlyplastic__(self) -> dict:
+        """Returns kwargs for ElasticPlastic constitutive law with no strain
+        hardening.
+        """
+        return {
+            'E': self.Es,
+            'fy': self.fyd(),
+            'eps_su': self.epsud(),
+        }
+
+    def __elasticplastic__(self) -> dict:
+        """Returns kwargs for ElasticPlastic constitutive law with strain
+        hardening.
+        """
+        Eh = (self.ftd() - self.fyd()) / (self.epsuk - self.epsyd)
```

Comment:
Should it be `self.epsud - self.epsyd` in the denominator?

Reply:
Same as before.